src/main/scala/diplomacy/Nodes.scala

```diff
@@ -611,8 +619,8 @@ class MixedNexusNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
   override def description = "nexus"
   protected[diplomacy] def resolveStar(iKnown: Int, oKnown: Int, iStars: Int, oStars: Int): (Int, Int) = {
     // a nexus treats :=* as a weak pointer
-    require (!outputRequiresInput || oKnown == 0 || iStars + iKnown != 0, s"$context has $oKnown required outputs and no possible inputs")
-    require (!inputRequiresOutput || iKnown == 0 || oStars + oKnown != 0, s"$context has $iKnown required inputs and no possible outputs")
+    require (!outputRequiresInput || oKnown == 0 || iStars + iKnown != 0, s"$context has $oKnown required outputs(${oBindings.map(_._2.name).mkString(",")}) and no possible inputs")
```

Comment:
How about 
`s"$context has $oKnown required outputs (named ${oBindings.map(_._2.name).mkString(",")}) and no inputs. At least one is required."`